GameData/RealismOverhaul/RO_DependentMods/RO_AJE.cfg

```diff
@@ -115,7 +115,7 @@
 
 	MODEL
 	{
-		model = VenStockRevamp/Part Bin/NewParts/JetEngines/HighBypassJet
+		model = VenStockRevamp/PartBin/NewParts/JetEngines/HighBypassJet
```

Comment:
Maybe support both old and new path using NEEDS? RO already does this in a few places https://github.com/KSP-RO/RealismOverhaul/search?q=partbin

(the path switch was actually to help us, because NEEDS doesn't work with spaces)

Reply:
Prefer not to carry forward supporting multiple versions of a mod in general, especially for a change like this.  Get newest, resolve problem.  Looks like I missed one reference when I did the earlier conversion.  (Or maybe something got added in after.  No big deal.)